src/frontend/Sider/Behandling/Inngangsvilkår/typer/stønadsperiode.ts

```diff
@@ -11,11 +12,5 @@ export interface Stønadsperiode extends Periode {
     id?: string;
     målgruppe: MålgruppeType | '';
     aktivitet: AktivitetType | '';
-    status?: StønadsperiodeStatus;
-}
-
-export enum StønadsperiodeStatus {
```

Comment:
Eneste refaktoreringen jeg gjorde i denne omgang var å endre navn på `StønadsperiodeStatus` -> `PeriodeStatus` og flytte den ut i egen fil, slik at den kunne gjenbrukes også for vedtaksperiodene.